app/assets/stylesheets/hacks/_header-tweaks.scss

```diff
@@ -40,3 +40,13 @@
     width: 40%;
   }
 }
+
+// Don't push the title to the 1/3 mark
+#global-header .header-wrapper .header-global .header-logo {
+  width: auto;
+}
+
+// Normal weight title like product pages
+.header-title {
+  font-weight: normal !important;
```

Comment:
How come we have to `!important` this? Could we perhaps use a slightly more specific selector to avoid it?